### PR TITLE
⤴️ Export CSVs

### PR DESCRIPTION
**Pull request by mermop**
This is for [this Trello card](https://trello.com/c/3KcxXX6Y/80-export-a-csv-of-tagged-submissions).

This includes:
- Save full tag number (eg 1.2.3.4) to the DB for ease of sorting
- Rake task to number unnumbered tags

### CODE REVIEW

**Comment by mermop** [comment]
Merging to get it onto staging this afternoon - would still appreciate feedback, can be addressed in the next PR @Rabid-Dan

File: app/controllers/consultations_controller.rb [hunk 1]
```diff
@@ -21,10 +21,28 @@ def create
...
+      end
+    end
+  end
+
   private
 
-  def find_consultation
-    Consultation.find(params[:id])
+  def consultation
+    @consultation ||= params[:id] ? Consultation.find(params[:id]) : Consultation.find(params[:consultation_id])
```
Comment (Rabid-Dan) [inline app/controllers/consultations_controller.rb]: could `Consultation.find(params[:id] || params[:consultation_id])` do the same thing?

File: app/controllers/consultations_controller.rb [hunk 2]
```diff
@@ -21,10 +21,28 @@ def create
     end
   end
 
+  def export # rubocop:disable Metrics/MethodLength
+    respond_to do |format|
+      format.csv do
+        case params[:type]
```
Comment (Rabid-Dan) [inline app/controllers/consultations_controller.rb]: maybe worth breaking this switch statement out into a little service object or a separate method or something in the future to make testing simpler.

**Comment by Rabid-Dan** [comment]
Keen to get some test coverage over those exporter classes when we can.

**Pull request merged**